### PR TITLE
Shift codejail CI to docker image

### PR DESCRIPTION
**Pull request by aht007**
The `codejail CI` jobs are essentially the last public-facing jobs on https://build.testeng.edx.org/ , which we really want to shut down or at least bring behind the VPN. This PR moves the CI jobs from Jenkins to Github Actions utilizing Docker Image and parsing `apparmor` profile using Github VM. Jenkins checks will be removed in a subsequent PR.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# Clone Codejail Repo
+ADD . ./codejail
+WORKDIR /codejail
+
+# Install codejail_sandbox sandbox dependencies
+RUN source $CODEJAIL_TEST_VENV/bin/activate && pip install -r requirements/sandbox.txt && deactivate
+
+# Install testing requirements
+RUN source /venv/bin/activate && pip install -r requirements/sandbox.txt && pip install -r requirements/testing.txt && deactivate
```
Comment (mraarif) [inline Dockerfile:39]: can we install these dependencies in one step?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: Codejail AppArmor CI
```
Comment (mraarif) [inline .github/workflows/ci.yml:1]: this should be named something like `codejail-ci` or `CI / tests`

**Comment by mraarif** [comment]
let's get this PR reviewed by the SRE team
cc: @nadeemshahzad

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,52 @@
+FROM ubuntu:focal
+SHELL ["/bin/bash", "-c"]
+
+# Install Codejail Packages
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y vim python3-virtualenv python3-pip
+RUN apt-get install -y sudo
```
[the remark this answers is not in the document]
Comment (aht007) [reply Dockerfile:7]: Yes we need to install sudo package as it is not available in docker by default

File: Dockerfile (hunk 1 above)
Comment (aht007) [reply Dockerfile:39]: Okay 👍

Comment (aht007) [reply Dockerfile:39]: These dependencies are being installed in two separate virtualenvs. One for Sandbox and other for the parent virtualenv

**Review by nadeemshahzad — APPROVED** [review]

**Review by mraarif — APPROVED** [review]

**Review by mraarif — APPROVED** [review]

**Pull request merged**